src/styles/_page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
```

Comment:
it is not a semantic class, if you call it `.page` it is better to set it on `<body>`, but it is on `<main>`, so call it `.main`